cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -157,9 +160,12 @@ def doc_type_mapping(val):
             return mapping(DOCUMENT_TYPE, val)
 
     for v in force_list(value):
-        val_a = doc_type_mapping(clean_val("a", v, str))
+        clean_val_a = clean_val("a", v, str)
+        val_a = doc_type_mapping(clean_val_a)
         val_b = doc_type_mapping(clean_val("b", v, str))
-
+        if ((key == "980__" or key == "690C_") and clean_val_a == "PERI") \
```

Comment:
in this case it should be a separate rule, moved to journals.py, because it is specific for journals, especially in terms of the values. 
For example in the data set of documents (not journals) we will have `PERI` value in the subfield `a`. From the point of view of the book it should raise UnexpectedValue. From the point of view of the journal this value is correct.

Reply:
I see, thanks! :+1: